dir.props

```diff
@@ -14,6 +14,11 @@
     <RoslynPackageName>Microsoft.Net.ToolsetCompilers</RoslynPackageName>
   </PropertyGroup>
 
+  <!-- MSBuild will set NuGetRuntimeIdentifier to be 'win' so we pre set it correctly here -->
+  <PropertyGroup>
+    <NuGetRuntimeIdentifier Condition="'$(NuGetRuntimeIdentifier)'==''">None</NuGetRuntimeIdentifier>
```

Comment:
the defaulting of 'win' is not comming from packageresolve.targets, it's comming from: `C:\Program Files (x86)\MSBuild\Microsoft\NuGet\Microsoft.NuGet.targets` which we can't really change. Actually packageresolve.targets are simply mimicking what Microsoft.NuGet.targets do.